test/unit/decorators/filter-comments.js

```diff
@@ -25,7 +25,7 @@ const assertComments = async (task, user, expected) => {
   const result = await decorator(settings)(task.model, user);
   const comments = result.comments;
   assert.equal(comments.length, expected.length);
-  assert.deepEqual(comments.map(c => c.comment), expected);
+  assert.deepEqual(comments.map(c => c.comment).sort(), expected.sort());
```

Comment:
This _shouldn't_ be necessary (and is probably bad) because comment ordering is important.

The history helper should be setting timestamps in a way that respects ordering, and the comments should be returned ordered by timestamp. If they're not then the underlying cause should be fixed rather than slapping a sort on them.
```suggestion
  assert.deepEqual(comments.map(c => c.comment), expected);
```

Reply:
yeah, bit of a lane swerve as they weren't actually running before. But I'll investigate why they are failing seeing as I made them run